operator/flags.go

```diff
@@ -32,12 +32,15 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	// AWS dedicated flags
-	flags.Int(option.AWSClientBurst, defaults.AWSClientBurst, "Burst value allowed for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientBurst)
-
-	flags.Float64(option.AWSClientQPSLimit, defaults.AWSClientQPSLimit, "Queries per second limit for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientQPSLimit)
+	flags.Int(option.AWSClientBurstDeprecated, defaults.IPAMAPIBurst, "")
+	flags.MarkDeprecated(option.AWSClientBurstDeprecated, fmt.Sprintf("please use --%s", option.IPAMAPIBurst))
```

Comment:
`option.BindEnv(option.AWSClientBurst)` seems to have been accidentally removed.

Reply:
I didn't add them because they did not exist < 1.8. Commit 0b762fed940e55d2df4e6f4eda96f7a8109470d7 added them for the first time.

Reply:
Ah, I didn't notice that, in that case yes.